.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+
+go:
+  - "1.10.x"
+
+install:
+  - ./scripts/install-protoc.sh
+
+script:
+- echo "Looking for protoc-gen-go ..." && find ~/ -name protoc-gen-go
```

Comment:
I forgot to cleanup this debug message. I'll push a change to get rid of this to the branch.